test/extended/setup.sh

```diff
@@ -345,6 +345,10 @@ readonly EXCLUDED_TESTS=(
 
 	# this tests the _kube_ downgrade. we don't support that.
 	"\[Feature:Downgrade\]"
+
+	# upstream flakes:
```

Comment:
We should be. That's what kube does. Otherwise you end up in test/merge "failure" purgatory.